app/main/__init__.py

```diff
@@ -19,6 +20,7 @@
 
 
 @main.before_request
+@flask_featureflags.is_active_feature('DIRECT_AWARD_PROJECTS')
```

Comment:
Given when this gets deployed, the feature flag will not be active for STAGING and PRODUCTION, does this mean that we are turning off `require_login` for those environments (or maybe 404ing)? Or have I misunderstood how the feature flags interact with this function?

https://github.com/trustrachel/Flask-FeatureFlags#controllersviews

Reply:
I'd certainly hope that they are entirely independent checks, i.e. this view is only accessible if the feature is enabled AND the user is logged in. If there's any doubt about that we'd better check!